**Мой первый расчёт**
Какова итоговая стоимость 5 футболок по цене 800 рублей каждая со скидкой 15%?

In [1]:
5 * 800 * 0.85

3400.0

Какова итоговая стоимость 15 шорт 
по цене 750 рублей каждая со скидкой 10%?

In [2]:
15 * 750 * 0.9

10125.0

При покупке 5 футболок по 800 руб. 
со скидкой 15% итоговая стоимость 
составила 3400 руб.

При покупке 15 шорт по 750 руб. 
со скидкой 10% итоговая стоимость 
составила 10125 руб.